src/UserStats/UserStatsRouteHandler.ts

```diff
@@ -0,0 +1,27 @@
+import * as UserStatsInteractor from './UserStatsInteractor';
+import { Request, Response, Router } from 'express';
+import { DataStore } from '../interfaces/DataStore';
+
+/**
+ * Initializes an express router with endpoints to get stats for a User
+ */
+export function initialize({ dataStore }: { dataStore: DataStore }) {
+  const router: Router = Router();
+  const getStats = async (req: Request, res: Response) => {
+    try {
+      const query = req.query;
+      const stats = await UserStatsInteractor.getStats({
+        dataStore,
+        query
+      });
+
+      res.status(200).send(stats);
+    } catch (e) {
+      console.error(e);
+      res.status(500).send(e);
+    }
+  };
+
+  router.route('').get(getStats);
```

Comment:
Not positive on this, but is the `.route('')` redundant since it doesn't modify the configuration of the where the caller places the route?

Could the following still work? Or is this a limitation of the Express API?

```suggestion
  router.get(getStats);
```